packages/models/test/fixtures/file-write-runner-config.ts

```diff
@@ -0,0 +1,17 @@
+import { platform } from 'os';
+import { AuditOutput, RunnerConfig } from '../../src';
+
+export function createFileWriteRunnerConfig(
```

Comment:
Maybe it would be better to rename this to `runnerConfig` (i.e. keep the name of the replaced function)? It's shorter and more in line with other mock naming conventions.

Reply:
OK, `echoRunnerConfig` is also a better name :slightly_smiling_face: